nemo_skills/prompt/template/llama3-instruct-nosys-translation.yaml

```diff
@@ -0,0 +1,30 @@
+# Prompt specification for the original Llama3-instruct model, but skipping the system message
+# This apparanetly is quite important to reproduce the math eval numbers
+
+# these tokens are always used to construct a prompt like this
+#
+#   single-turn:
+#     <text_begin><system_begin>{system}<system_end><user_begin>{user}<user_end><assistant_begin>{generation}
+#   multi-turn:
+#     <text_begin><system_begin>{system}<system_end><user_begin>{user1}<user_end><assistant_begin>{assistant1}<assistant_end>...
+#     <user_begin>{userN}<user_end><assistant_begin>{generation}
+
+text_begin: "<|begin_of_text|>"
+
+system_begin: ""
+system_end: ""
+
+user_begin: "<|start_header_id|>user<|end_header_id|>\n\n"
+user_end: "<|eot_id|>"
+
+assistant_begin: "<|start_header_id|>assistant<|end_header_id|>\n\n"
+assistant_end: "<|eot_id|>"
+
+stop_phrases: ["<|eot_id|>", "sorry"]
```

Comment:
Let's maybe make a new extra_stop_phrases parameter to the generate function, so that we don't need to create a new config just for that? Also please directly create that PR to main